config/manager/kustomization.yaml

```diff
@@ -1,2 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
 resources:
 - manager.yaml
+images:
+- name: controller
+  newName: quay.io/omergi/overlay-network-controller
```

Comment:
Please change according to the new proposed image name.

Reply:
Done